app/retail/views.py

```diff
@@ -1030,6 +1029,12 @@ def create_status_update(request):
             response['message'] = 'Bad Request'
             logger.error('Status Update error - Error: (%s) - Handle: (%s)', e, profile.handle if profile else '')
             return JsonResponse(response, status=400)
+    else:
+        response['status'] = 401
+        response['message'] = 'Not logged in!'
+        print('here', response)
+        return JsonResponse(status=response['status'], data={'status': 401,'message':response['message']})
```

Comment:
can't we just sent return JsonResponse(response, status=401) ?

Reply:
how it works is status 401 http status data.status json pattern

Reply:
Yes that cleaner